src/Components/LoginPage/LoginPage.js

```diff
@@ -20,6 +22,13 @@ const LoginPage = () => {
       [event.target.name]: event.target.value
     }))
   }
+
+  useEffect(() => {
+    if(data) {
+      console.log('data', data.users)
+      setAllUsers(data.users)
+    }
+  }, [data])
```

Comment:
note for the future: I'm not sure if we'll need to set the data returned by the `GET_ALL_USERS` query to state. I was reading online it's possible to directly access `data` from the destructured `useQuery` response. This works for now though.